src/pydata_sphinx_theme/theme/pydata_sphinx_theme/layout.html

```diff
@@ -27,7 +27,7 @@
                                    or not theme_secondary_sidebar_items %}
 
 {%- block css %}
-  <script>
+  <script data-cfasync="false">
```

Comment:
Can you just add a short comment explanation and link to this PR so that future devs have context for this?